Woof/Woof/Helper/DataTrasnformer/DataTransformer.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+/// A utility for converting and formatting data into a format convenient for the work of the application
+/// and displaying data to the user.
+enum DataTransformer {
+    /// Returns full name by combining the given name and surname.
+    ///
+    /// Examples of usage:
+    /// ```
+    /// let fullName = fullName(name: "Kate", surname: "Peterson")
+    /// print(fullName) // "Kate Peterson"
+    ///
+    /// let fullName = fullName(name: "KaTe", surname: "PeTErsoN")
+    /// print(fullName) // "Kate Peterson"
+    ///
+    /// let fullName = fullName(name: "KATE", surname: "PETERSON")
+    /// print(fullName) // "Kate Peterson"
+    /// ```
+    ///
+    /// - Parameters:
+    ///   - surname: The surname of the person.
+    ///   - name: The name of the person.
+    /// - Returns: The full name in the format "Name Surname".
+    static func fullName(name: String, surname: String) -> String {
```

Comment:
nit for now: should the `fullName` be a computed property within the `Sitter` model?